pydatastructs/graphs/algorithms.py

```diff
@@ -785,19 +807,19 @@ def all_pair_shortest_paths(graph: Graph, algorithm: str) -> tuple:
     Examples
     ========
 
-    >>> from pydatastructs import Graph, AdjacencyListGraphNode
-    >>> from pydatastructs import all_pair_shortest_paths
-    >>> V1 = AdjacencyListGraphNode("V1")
-    >>> V2 = AdjacencyListGraphNode("V2")
-    >>> V3 = AdjacencyListGraphNode("V3")
-    >>> G = Graph(V1, V2, V3)
-    >>> G.add_edge('V2', 'V3', 10)
-    >>> G.add_edge('V1', 'V2', 11)
-    >>> G.add_edge('V3', 'V1', 5)
-    >>> dist, _ = all_pair_shortest_paths(G, 'floyd_warshall')
-    >>> dist['V1']['V3']
+    from pydatastructs import Graph, AdjacencyListGraphNode
+    from pydatastructs import all_pair_shortest_paths
+    V1 = AdjacencyListGraphNode("V1")
+    V2 = AdjacencyListGraphNode("V2")
+    V3 = AdjacencyListGraphNode("V3")
+    G = Graph(V1, V2, V3)
+    G.add_edge('V2', 'V3', 10)
+    G.add_edge('V1', 'V2', 11)
+    G.add_edge('V3', 'V1', 5)
+    dist, _ = all_pair_shortest_paths(G, 'floyd_warshall')
+    dist['V1']['V3']
```

Comment:
Why are `>>>` removed?

Reply:
Please address this concern. Merge back master to your branch, apply the flake8 formatting again and I will merge this.

Reply:
@hpnightowl

Reply:
sure today will do sorry for delay !